app/helpers/reactjs_helper.rb

```diff
@@ -20,9 +20,7 @@ def select_requested_plugins(webpacked_plugin_ids, plugin_names)
   end
 
   def all_webpacked_plugins
```

Comment:
I'm wondering if this function should be removed given it's no real additional value.

Reply:
sure, i'll remove it.